RELEASING.md

```diff
@@ -47,12 +67,19 @@ After preparing the machine follow the below steps
     ```
     git push origin master
     git push origin --tags
+
     ```
 
-After pushing the tags, travis should automatically start building the tags and push the artifact to the maven central. To monitor the build process go to [travis-ci](https://travis-ci.org/) and navigate to beadledom project.
+If at anytime the release need to be stopped. Cancel the maven commands and run the below command
 
-If at anytime the released need to be stopped. Cancel the maven commands and run the below command
+```
+mvn release:rollback
+```
 
-    ```
-    mvn release:rollback
-    ```
+Once the master branch and tags are pushed the following happens.
+
+* Travis kicks off a new build for released tag pushes the artifact to Maven central. It roughly takes about 2 hours for the artifacts to sync with the maven central.
```

Comment:
NIT: can we rephrase the first sentence?
maybe: A new build is run by Travis to push the released artifact to Maven central.